app/views/deploymentSitesDashboard.scala.html

```diff
@@ -44,17 +47,28 @@
                 </div>
             </div>
         </div>
+
+        <div class="cta-section">
+            <div class="cta-content">
+                <img src='@assets.path("images/logos/ProjectSidewalkLogo_WithCurvedText_3600x3600.png")' alt="@Messages("cities.dashboard.cta.logo.alt")" class="cta-logo"/>
+
+                <h2 class="cta-title">@Messages("cities.dashboard.cta.title")</h2>
+                <p class="cta-text">@Html(Messages("cities.dashboard.cta.content"))</p>
+                <a href="https://github.com/ProjectSidewalk/SidewalkWebpage/wiki" class="cta-button" target="_blank" rel="noopener noreferrer">
```

Comment:
Should we be linking here instead of the wiki like we do on the landing page?
https://accessiblecommunities.org/use-project-sidewalk/
<img width="927" height="174" alt="image" src="https://github.com/user-attachments/assets/da7c4251-28e4-4598-8d6f-c396be231803" />